lib/tiler.ts

```diff
@@ -77,8 +79,9 @@ export async function TileServer<T>({
           query: filtersQuery,
           debug,
           zoomToDistance,
+          getBaseQuery,
         });
-        const result = await pool.query(query.sql, query.values);
```

Comment:
even without slonik, values should be used instead of directly putting SQL strings in the query